desktop/ui/src/test/java/org/datacleaner/job/runner/RowProcessingConsumerSorterTest.java

```diff
@@ -76,7 +77,7 @@ public void testCreateProcessOrderedConsumerListWithMergedOutcomes() throws Exce
         fjb1.setName("fjb1");
 
         // 2: trim (depends on filter)
-        TransformerComponentBuilder<TransformerMock> tjb1 = ajb.addTransformer(TransformerMock.class);
+        TransformerComponentBuilder<MockTransformer> tjb1 = ajb.addTransformer(MockTransformer.class);
```

Comment:
I did look if it was used still, and still 2 unittests use it... But probably it can be replaced easily if we want to.